src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -35,14 +39,19 @@ public void SelectOutputFolder()
          _outputFolderDTO.FolderPath = outputFolder;
       }
 
+      public override void ViewChanged()
+      {
+         _view.OkEnabled = _view.HasError;
+      }
+
       public void Abort()
       {
-         throw new System.NotImplementedException();
+         _batchStarterTask.StopValidation(_process);
```

Comment:
Why do you want to manage that as async? There is no need since the process is completely async anyway no?